package.json

```diff
@@ -3,34 +3,39 @@
   "version": "1.0.0",
   "description": "MongoDB connection for Relay",
   "main": "lib/index.js",
-  "repository" : {
-    "type" : "git",
-    "url" : "https://github.com/mikberg/relay-mongodb-connection.git"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mikberg/relay-mongodb-connection.git"
   },
   "scripts": {
-    "compile": "mkdir -p lib && babel index.js --out-file lib/index.js",
+    "compile": "webpack --colors --progress -p",
```

Comment:
why use webpack? If anything, I think we should aim to remove a build step alltogether